docs/Docker.md

```diff
@@ -0,0 +1,33 @@
+# Docker usage with phpsu
+
+You can use phpsu's Docker image if you like.
+
+With the Docker image, you have some advantages, but also some limitations.
+
+## Advantages, or why should I use the Docker image:
+
+- you don't need to use composer it is all bundled within the Docker image
+
+## Limitations, or why shouldn't I use the Docker image?:
+
+- you should understand something about dockers networking
+- you local composer autoloading is not working, as phpsu is installed globally inside the container.
+- if needed: you need to give the container all your necessary Environment variables
+
+## Basic Usage
+
+The Image has phpsu and all it's necessary dependencies installed.
+There's just a few things you need to get started:
+- your ssh key needs to be inside the container.  
```

Comment:
that's not exactly a requirement since you could use it with just passwords from the config (in some cases a ssh-key is not possible)

Reply:
for a database sync i dont need an ssh connection, if the port is public e.g.
And there is a workaround (not included [yet]) in this package to do ssh with password instead of ssh-key